src/Blocks/BlockHandler.h

```diff
@@ -110,8 +110,11 @@ class cBlockHandler
 
 	/** Checks if the player can build "inside" this block.
 	For example blocks placed "on" snow will be placed at the same position. So: Snow ignores Build collision
+	@param a_Pos Position of the block
+	@param a_Player Player trying to build on the block
+	@param a_Meta Meta value of the block
```

Comment:
Just the meta.
Just curious, why not make the position parameter `const`? It could give the compiler more room for optimizations.